src/components/RadioGroup/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import { commonLabelClasses, commonIconClasses, inputClasses, disabledClasses } from '../Checkbox'
+
+const radioClasses = 'mb2 mh2 eh-radio'
+const radioIconClasses = `${commonIconClasses} br-pill eh-radio-icon`
+
+class RadioButtonGroup extends Component {
+  static propTypes = {
+    optionList: React.PropTypes.array.isRequired
```

Comment:
@trevordmiller Awesome, thanks for the example! I forgot about the `shape` prop.